site/content/docs/main/aggregator-permissions.md

```diff
@@ -0,0 +1,27 @@
+# Aggregator Permissions
+
+Through `sonobuoy gen` command you can customize multiple aspects of the run in your cluster. In these multiple aspects you can also customize aggregator permissions of a user in your cluster. These permissions act as limitations, it determines that in what tasks a user can take part in the run. We can customize this aspect of the run through `aggregator-permissions` flag.
+
+- Run `sonobuoy gen` without changing aggregator permissions, value of parameter `AggregatorPermissions` is `clusterAdmin`in sonobuoy config.
+
+![aggregator-permissions1](img/aggregator-permissions1.png)
```

Comment:
I'd remove these screenshots; I dont think it adds anything meaningful to the users understanding to see these and it gives the false impression that the config value shown is the only thing that changes.